content/guides/getting-started/devs-vs-users.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Who are you Testing for?
+---
+
+When testing components, you should first realize that you have TWO kinds users:
```

Comment:
```suggestion
When testing components, you should first realize that you have TWO kinds of users:
```

"you should first realize" comes off a little condescending to me. How about the following?

```
When testing components, there are two kinds of users:
```